app/download/views.py

```diff
@@ -77,6 +120,49 @@ def download_document_b64(service_id, document_id):
     except ValueError:
         abort(404)
 
+    try:
+        check_scan_verdict(service_id, document_id, sending_method)
+    except (MaliciousContentError, SuspiciousContentError) as e:
+        current_app.logger.info(
+            "Malicious content detected, refused to download document: {}".format(e),
+            extra={
+                "service_id": service_id,
+                "document_id": document_id,
+            },
+        )
+        return jsonify(error=str(e)), MALICIOUS_CONTENT_ERROR_CODE
```

Comment:
```suggestion
        abort(404)
```

Since this is what a user would see in their web browser if they click on a link type attachment that is malware, we don't need to give any extra info here. Of course this should not happen because notification-api will not send an email containing malware, but it's a safety check.